Tools/setup/arch.sh

```diff
@@ -119,6 +123,43 @@ if [[ $INSTALL_SIM == "true" ]]; then
 		ant \
 		jdk8-openjdk \
 		;
+
+	# Gazebo setup
+	if [[ $INSTALL_GAZEBO == "true" ]]; then
+		echo
+		echo "Installing gazebo and dependencies for PX4 simulation"
+
+		# PX4 gazebo simulation dependencies
+		sudo pacman -S --noconfirm --needed \
+			eigen3 \
+			hdf5 \
+			opencv \
+			protobuf \
+			vtk \
+			yay \
+			;
+
+		# enable multicore gazebo compilation
+		sudo sed -i '/MAKEFLAGS=/c\MAKEFLAGS="-j'$(($(grep -c processor /proc/cpuinfo)+2))'"' /etc/makepkg.conf
```

Comment:
Haha, and that's why I don't run these scripts :confounded:.

Reply:
Well now you can 😄

Reply:
What would be a better way to get reasonable compilation times?

Reply:
Ok, so my opinion is that if you're using Arch that's probably something you should figure out yourself.

Reply:
Arch == no convenience allowed? :stuck_out_tongue_closed_eyes:

Reply:
Yer not sure :thinking: 
https://wiki.archlinux.org/index.php/Arch_Linux#Principles